rshim.spec.in

```diff
@@ -57,6 +57,8 @@ make
 %{__install} -m 0644 etc/rshim.conf %{buildroot}%{_sysconfdir}
 %{__install} -m 0755 scripts/bfb-install %{buildroot}%{_sbindir}
 %{__install} -m 0755 scripts/bf-reg %{buildroot}%{_sbindir}
+mkdir -p %{buildroot}%{_datadir}/selinux/packages
+%{__install} -m 0644 scripts/rshim-fix.te %{buildroot}%{_datadir}/selinux/packages/rshim-fix.te
```

Comment:
Any reason not to use rshim.te instead of rshim-fix.te?

Reply:
rshim.te is distributed in SELinux official repo (github.com:fedora-selinux/selinux-policy.git) and installed in the system already.